src/components/LoginPopup/LoginPopUp.css

```diff
@@ -5,11 +5,12 @@
   left: 0;
   width: 100%;
   height: 100%;
-  background-color: rgba(0, 0, 0, 0.5); /* semi-transparent background */
```

Comment:
maybe 0.7, khater lktiba currently tetra wra el popup (not what we want)
![image](https://github.com/Fabeure/Studyfied/assets/75645024/37751383-ce78-4ff8-830d-4ca492431d56)